torch/_dynamo/resume_execution.py

```diff
@@ -31,6 +31,76 @@ class ReenterWith:
     stack_index: int = None
     target_values: Optional[Tuple] = None
 
+    # If we do not want to destroy the stack, we can do the same thing as a
+    # `SETUP_WITH` block, only that we store the context manager in a local_symbol
+    def try_except(self, code_options, cleanup: List[Instruction]):
+        load_args = []
+        if self.target_values:
+            load_args = [
+                create_instruction(
+                    "LOAD_CONST",
+                    PyCodegen.get_const_index(code_options, val),
+                    val,
+                )
+                for val in self.target_values
+            ]
+        ctx_name = f"___context_manager_{self.stack_index}"
```

Comment:
This can be duplicated, can you use this helper function?
https://github.com/pytorch/pytorch/blob/34a7c79eac1b0743c76043b81cd2b32326cb9035/torch/_dynamo/bytecode_transformation.py#L580-L581